integration-tests/jail/run_test.sh

```diff
@@ -12,11 +12,5 @@ echo "docker compose ok"
 nix-shell ./jail.nix  --run "export PASSPHRASE=1 && python3 ../bot/jail_test.py"
```

Comment:
Can be written as `PASSPHRASE=1 python3 ../bot/jail_test.py`

Reply:
ok